src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition:
+    transform 300ms ease,
+    background-color 300ms ease;
```

Comment:
The `background-color` property is not specified for the `.card` class. Therefore, the transition for `background-color` seems to be unnecessary.